workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-filter/package.json

```diff
@@ -1,19 +1,21 @@
 {
 	"dependencies": {
-		"@liferay/js-api": "0.6.0-pre.0",
-		"serve": "^14.2.0"
+		"@liferay/js-api": "0.6.0-pre.0"
 	},
 	"description": "Liferay Sample FDS Filter",
 	"devDependencies": {
-		"typescript": "^5.0.4"
+		"ts-loader": "9.5.1",
+		"typescript": "^5.0.4",
+		"webpack": "5.90.1",
+		"webpack-cli": "5.1.4",
+		"webpack-dev-server": "4.15.1"
 	},
 	"main": "./src/index.ts",
 	"name": "@liferay/liferay-sample-fds-filter",
 	"scripts": {
-		"build": "tsc",
-		"start": "PORT=3000 serve --cors",
+		"build": "webpack",
+		"start": "webpack serve",
 		"watch": "tsc --watch"
```

Comment:
what about `watch`? shall we use webpack here too?

Reply:
No, because `watch` is to recompile every time you change something in the IDE, so there's no `watch` concept for webpack...